clang/lib/AST/StmtPrinter.cpp

```diff
@@ -1077,6 +1077,16 @@ void StmtPrinter::VisitObjCSubscriptRefExpr(ObjCSubscriptRefExpr *Node) {
   OS << "]";
 }
 
+void StmtPrinter::VisitUniqueStableNameExpr(UniqueStableNameExpr *Node) {
+  // FIXME: Is this the correct thing?
```

Comment:
It looks like what I'd expect.

Reply:
Btw, I think you need something similar in JSONNodeDumper.cpp for dumping to JSON, and TextNodeDumper.cpp for dumping to text.

Reply:
Good tip!  I didn't think about the dumpers, and it didn't diagnose I'd missed those places.

Reply:
Dumpers are mostly a best-effort sort of thing, but they're trivial to support and so I usually try to nerd snipe people into doing them. :-D